Bug: Bitbucket 5.15.0 error 500x on a few pages

**Describe the bug**
Http Error 500 when accessing Bitbucket User Directories.

**To Reproduce**
Steps to reproduce the behavior:
1. Bitbucket 5.15.0 (Didn't test on production which is a version lower)
2. APM latest version
3. Go to Bitbucket URI : /plugins/servlet/embedded-crowd/directories/list
4. See error

**Expected behavior**
Get the User Directories page without any error

**Debug logs**
Attach your debug logs.
See the [documentation](https://www.elastic.co/guide/en/apm/agent/java/current/trouble-shooting.html#trouble-shooting-logging)
about how to enable debug logging.

<details>
<summary>Click to expand</summary>

```
2018-11-26 08:58:18,287 ERROR [http-nio-7990-exec-7]  o.a.c.c.C.[.[.[/].[plugins] Servlet.service() for servlet [plugins] in context with path [] threw exception [Servlet execution threw an exception] with root cause
java.lang.ClassNotFoundException: co.elastic.apm.servlet.ServletApiAdvice not found by org.apache.servicemix.bundles.spring-webmvc [8]
        at org.apache.felix.framework.BundleWiringImpl.findClassOrResourceByDelegation(BundleWiringImpl.java:1532)
        at com.atlassian.applinks.core.rest.context.ContextFilter.doFilter(ContextFilter.java:24)
        at com.atlassian.applinks.core.rest.context.ContextFilter.doFilter(ContextFilter.java:24)
        at com.atlassian.applinks.core.rest.context.ContextFilter.doFilter(ContextFilter.java:24)
        at com.atlassian.applinks.core.rest.context.ContextFilter.doFilter(ContextFilter.java:24)
        at com.atlassian.applinks.core.rest.context.ContextFilter.doFilter(ContextFilter.java:24)
        at com.atlassian.analytics.client.filter.UniversalAnalyticsFilter.doFilter(UniversalAnalyticsFilter.java:92)
        at com.atlassian.analytics.client.filter.AbstractHttpFilter.doFilter(AbstractHttpFilter.java:39)
        at com.atlassian.stash.internal.spring.lifecycle.LifecycleJohnsonServletFilterModuleContainerFilter.doFilter(LifecycleJohnsonServletFilterModuleContainerFilter.java:42)
        at com.opensymphony.sitemesh.webapp.SiteMeshFilter.obtainContent(SiteMeshFilter.java:181)
        at com.opensymphony.sitemesh.webapp.SiteMeshFilter.doFilter(SiteMeshFilter.java:85)
        at com.atlassian.plugin.connect.plugin.auth.scope.ApiScopingFilter.doFilter(ApiScopingFilter.java:81)
        at com.atlassian.stash.internal.spring.lifecycle.LifecycleJohnsonServletFilterModuleContainerFilter.doFilter(LifecycleJohnsonServletFilterModuleContainerFilter.java:42)
        at com.atlassian.stash.internal.spring.security.StashAuthenticationFilter.doFilter(StashAuthenticationFilter.java:85)
        at com.atlassian.stash.internal.web.auth.BeforeLoginPluginAuthenticationFilter.doInsideSpringSecurityChain(BeforeLoginPluginAuthenticationFilter.java:112)
        at com.atlassian.stash.internal.web.auth.BeforeLoginPluginAuthenticationFilter.doFilter(BeforeLoginPluginAuthenticationFilter.java:75)
        at com.atlassian.security.auth.trustedapps.filter.TrustedApplicationsFilter.doFilter(TrustedApplicationsFilter.java:94)
        at com.atlassian.oauth.serviceprovider.internal.servlet.OAuthFilter.doFilter(OAuthFilter.java:67)
        at com.atlassian.stash.internal.spring.lifecycle.LifecycleJohnsonServletFilterModuleContainerFilter.doFilter(LifecycleJohnsonServletFilterModuleContainerFilter.java:42)
        at com.atlassian.plugin.connect.plugin.auth.oauth2.DefaultSalAuthenticationFilter.doFilter(DefaultSalAuthenticationFilter.java:69)
        at com.atlassian.plugin.connect.plugin.auth.user.ThreeLeggedAuthFilter.doFilter(ThreeLeggedAuthFilter.java:109)
        at com.atlassian.jwt.internal.servlet.JwtAuthFilter.doFilter(JwtAuthFilter.java:32)
        at com.atlassian.analytics.client.filter.DefaultAnalyticsFilter.doFilter(DefaultAnalyticsFilter.java:38)
        at com.atlassian.analytics.client.filter.AbstractHttpFilter.doFilter(AbstractHttpFilter.java:39)
        at com.atlassian.bitbucket.internal.scm.git.lfs.servlet.filter.GitLfsLockingFilter.doFilter(GitLfsLockingFilter.java:50)
        at com.atlassian.stash.internal.spring.lifecycle.LifecycleJohnsonServletFilterModuleContainerFilter.doFilter(LifecycleJohnsonServletFilterModuleContainerFilter.java:42)
        at com.atlassian.stash.internal.web.auth.BeforeLoginPluginAuthenticationFilter.doBeforeBeforeLoginFilters(BeforeLoginPluginAuthenticationFilter.java:90)
        at com.atlassian.stash.internal.web.auth.BeforeLoginPluginAuthenticationFilter.doFilter(BeforeLoginPluginAuthenticationFilter.java:73)
        at com.atlassian.stash.internal.request.DefaultRequestManager.doAsRequest(DefaultRequestManager.java:87)
        at com.atlassian.stash.internal.hazelcast.ConfigurableWebFilter.doFilter(ConfigurableWebFilter.java:38)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.lang.Thread.run(Thread.java:748)
        ... 232 frames trimmed

2018-11-26 08:58:18,291 ERROR [http-nio-7990-exec-7] @1BKCLG4x538x38428x0 1lonpf8 10.185.116.60,127.0.0.1 "GET /mvc/error500 HTTP/1.1" c.a.s.i.web.ErrorPageController There was an unhandled exception loading [/plugins/servlet/embedded-crowd/directories/list]
javax.servlet.ServletException: Servlet execution threw an exception
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:238)
        at com.atlassian.applinks.core.rest.context.ContextFilter.doFilter(ContextFilter.java:24)
        at com.atlassian.applinks.core.rest.context.ContextFilter.doFilter(ContextFilter.java:24)
        at com.atlassian.applinks.core.rest.context.ContextFilter.doFilter(ContextFilter.java:24)
        at com.atlassian.applinks.core.rest.context.ContextFilter.doFilter(ContextFilter.java:24)
        at com.atlassian.applinks.core.rest.context.ContextFilter.doFilter(ContextFilter.java:24)
        at com.atlassian.analytics.client.filter.UniversalAnalyticsFilter.doFilter(UniversalAnalyticsFilter.java:92)
        at com.atlassian.analytics.client.filter.AbstractHttpFilter.doFilter(AbstractHttpFilter.java:39)
        at com.atlassian.stash.internal.spring.lifecycle.LifecycleJohnsonServletFilterModuleContainerFilter.doFilter(LifecycleJohnsonServletFilterModuleContainerFilter.java:42)
        at com.opensymphony.sitemesh.webapp.SiteMeshFilter.obtainContent(SiteMeshFilter.java:181)
        at com.opensymphony.sitemesh.webapp.SiteMeshFilter.doFilter(SiteMeshFilter.java:85)
        at com.atlassian.plugin.connect.plugin.auth.scope.ApiScopingFilter.doFilter(ApiScopingFilter.java:81)
        at com.atlassian.stash.internal.spring.lifecycle.LifecycleJohnsonServletFilterModuleContainerFilter.doFilter(LifecycleJohnsonServletFilterModuleContainerFilter.java:42)
        at com.atlassian.stash.internal.spring.security.StashAuthenticationFilter.doFilter(StashAuthenticationFilter.java:85)
        at com.atlassian.stash.internal.web.auth.BeforeLoginPluginAuthenticationFilter.doInsideSpringSecurityChain(BeforeLoginPluginAuthenticationFilter.java:112)
        at com.atlassian.stash.internal.web.auth.BeforeLoginPluginAuthenticationFilter.doFilter(BeforeLoginPluginAuthenticationFilter.java:75)
        at com.atlassian.security.auth.trustedapps.filter.TrustedApplicationsFilter.doFilter(TrustedApplicationsFilter.java:94)
        at com.atlassian.oauth.serviceprovider.internal.servlet.OAuthFilter.doFilter(OAuthFilter.java:67)
        at com.atlassian.stash.internal.spring.lifecycle.LifecycleJohnsonServletFilterModuleContainerFilter.doFilter(LifecycleJohnsonServletFilterModuleContainerFilter.java:42)
        at com.atlassian.plugin.connect.plugin.auth.oauth2.DefaultSalAuthenticationFilter.doFilter(DefaultSalAuthenticationFilter.java:69)
        at com.atlassian.plugin.connect.plugin.auth.user.ThreeLeggedAuthFilter.doFilter(ThreeLeggedAuthFilter.java:109)
        at com.atlassian.jwt.internal.servlet.JwtAuthFilter.doFilter(JwtAuthFilter.java:32)
        at com.atlassian.analytics.client.filter.DefaultAnalyticsFilter.doFilter(DefaultAnalyticsFilter.java:38)
        at com.atlassian.analytics.client.filter.AbstractHttpFilter.doFilter(AbstractHttpFilter.java:39)
        at com.atlassian.bitbucket.internal.scm.git.lfs.servlet.filter.GitLfsLockingFilter.doFilter(GitLfsLockingFilter.java:50)
        at com.atlassian.stash.internal.spring.lifecycle.LifecycleJohnsonServletFilterModuleContainerFilter.doFilter(LifecycleJohnsonServletFilterModuleContainerFilter.java:42)
        at com.atlassian.stash.internal.web.auth.BeforeLoginPluginAuthenticationFilter.doBeforeBeforeLoginFilters(BeforeLoginPluginAuthenticationFilter.java:90)
        at com.atlassian.stash.internal.web.auth.BeforeLoginPluginAuthenticationFilter.doFilter(BeforeLoginPluginAuthenticationFilter.java:73)
        at com.atlassian.stash.internal.request.DefaultRequestManager.doAsRequest(DefaultRequestManager.java:87)
        at com.atlassian.stash.internal.hazelcast.ConfigurableWebFilter.doFilter(ConfigurableWebFilter.java:38)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.lang.Thread.run(Thread.java:748)
        ... 222 frames trimmed
Caused by: java.lang.NoClassDefFoundError: co/elastic/apm/servlet/ServletApiAdvice
        at org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:841)
        ... 32 common frames omitted
Caused by: java.lang.ClassNotFoundException: co.elastic.apm.servlet.ServletApiAdvice not found by org.apache.servicemix.bundles.spring-webmvc [8]
        at org.apache.felix.framework.BundleWiringImpl.findClassOrResourceByDelegation(BundleWiringImpl.java:1532)
        ... 32 common frames omitted

```
</details>

The first error repeats itself around 15times.

No it didn't work. 

The error is:
```
A fatal error has occurred
The following problems occurred, which prevents Atlassian Bitbucket from starting:
Required plugin com.atlassian.activeobjects.activeobjects-plugin has failed to start
Required plugin com.atlassian.applinks.applinks-plugin has failed to start
Required plugin com.atlassian.atlassian-failure-cache-plugin has failed to start
Required plugin com.atlassian.auiplugin has failed to start
Required plugin com.atlassian.bitbucket.server.bitbucket-nav-links has failed to start
Required plugin com.atlassian.bitbucket.server.bitbucket-rest has failed to start
Required plugin com.atlassian.bitbucket.server.bitbucket-sal has failed to start
Required plugin com.atlassian.bitbucket.server.bitbucket-velocity-helper has failed to start
Required plugin com.atlassian.crowd.embedded.admin has failed to start
Required plugin com.atlassian.markup.atlassian-markup-renderer-plugin has failed to start
Required plugin com.atlassian.oauth.admin has failed to start
Required plugin com.atlassian.oauth.atlassian-oauth-service-provider-spi has failed to start
Required plugin com.atlassian.oauth.consumer has failed to start
Required plugin com.atlassian.oauth.consumer.sal has failed to start
Required plugin com.atlassian.oauth.serviceprovider has failed to start
Required plugin com.atlassian.oauth.serviceprovider.sal has failed to start
Required plugin com.atlassian.plugins.atlassian-client-resource has failed to start
Required plugin com.atlassian.plugins.atlassian-nav-links-plugin has failed to start
Required plugin com.atlassian.plugins.atlassian-plugins-webresource-plugin has failed to start
Required plugin com.atlassian.plugins.atlassian-plugins-webresource-rest has failed to start
Required plugin com.atlassian.plugins.less-transformer-plugin has failed to start
Required plugin com.atlassian.plugins.rest.atlassian-rest-module has failed to start
Required plugin com.atlassian.soy.soy-template-plugin has failed to start
Required plugin com.atlassian.stash.ssh-plugin has failed to start
Required plugin com.atlassian.templaterenderer.api has failed to start
Required plugin com.atlassian.templaterenderer.atlassian-template-renderer-velocity1.6-plugin has failed to start
Required plugin com.atlassian.troubleshooting.plugin-bitbucket has failed to start
Required plugin com.atlassian.upm.atlassian-universal-plugin-manager-plugin has failed to start
```
And in the logs:
```
2018-12-21 08:23:17,567 ERROR [ThreadPoolAsyncTaskExecutor::Thread 55]  o.e.g.b.e.i.d.s.DependencyWaiterApplicationContextExecutor Unable to create application context for [com.atlassian.bitbucket.server.bitbucket-jira-development-integration], unsatisfied dependencies: none
java.lang.NoClassDefFoundError: sun/reflect/ConstructorAccessorImpl
        at sun.misc.Unsafe.defineClass(Native Method)
        at java.security.AccessController.doPrivileged(Native Method)
        at org.eclipse.gemini.blueprint.context.support.TrackingUtil.getService(TrackingUtil.java:137)
        at org.eclipse.gemini.blueprint.context.support.OsgiBundleXmlApplicationContext.lookupNamespaceHandlerResolver(OsgiBundleXmlApplicationContext.java:269)
        at org.eclipse.gemini.blueprint.context.support.OsgiBundleXmlApplicationContext.createNamespaceHandlerResolver(OsgiBundleXmlApplicationContext.java:229)
        at org.eclipse.gemini.blueprint.context.support.OsgiBundleXmlApplicationContext.loadBeanDefinitions(OsgiBundleXmlApplicationContext.java:130)
        at org.springframework.context.support.AbstractRefreshableApplicationContext.refreshBeanFactory(AbstractRefreshableApplicationContext.java:129)
        at org.springframework.context.support.AbstractApplicationContext.obtainFreshBeanFactory(AbstractApplicationContext.java:614)
        at org.eclipse.gemini.blueprint.context.support.AbstractDelegatedExecutionApplicationContext.access$800(AbstractDelegatedExecutionApplicationContext.java:60)
        at org.eclipse.gemini.blueprint.context.support.AbstractDelegatedExecutionApplicationContext$3.run(AbstractDelegatedExecutionApplicationContext.java:242)
        at org.eclipse.gemini.blueprint.util.internal.PrivilegedUtils.executeWithCustomTCCL(PrivilegedUtils.java:85)
        at org.eclipse.gemini.blueprint.context.support.AbstractDelegatedExecutionApplicationContext.startRefresh(AbstractDelegatedExecutionApplicationContext.java:220)
        at org.eclipse.gemini.blueprint.extender.internal.dependencies.startup.DependencyWaiterApplicationContextExecutor.stageOne(DependencyWaiterApplicationContextExecutor.java:224)
        at org.eclipse.gemini.blueprint.extender.internal.dependencies.startup.DependencyWaiterApplicationContextExecutor.refresh(DependencyWaiterApplicationContextExecutor.java:177)
        at org.eclipse.gemini.blueprint.context.support.AbstractDelegatedExecutionApplicationContext.refresh(AbstractDelegatedExecutionApplicationContext.java:157)
        at org.eclipse.gemini.blueprint.extender.internal.activator.LifecycleManager$1.run(LifecycleManager.java:207)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.lang.Thread.run(Thread.java:748)
        ... 10 frames trimmed
Caused by: java.lang.ClassNotFoundException: sun.reflect.ConstructorAccessorImpl not found by org.apache.servicemix.bundles.spring-beans [10]
        at org.apache.felix.framework.BundleWiringImpl.findClassOrResourceByDelegation(BundleWiringImpl.java:1532)
        ... 19 common frames omitted
2018-12-21 08:23:17,583 INFO  [ThreadPoolAsyncTaskExecutor::Thread 58]  o.e.g.b.e.i.s.ExtenderConfiguration Application context successfully refreshed (NonValidatingOsgiBundleXmlApplicationContext(bundle=com.atlassian.bitbucket.server.bitbucket-crowd-spi, config=osgibundle:/META-INF/spring/*.xml))
2018-12-21 08:23:17,584 ERROR [ThreadPoolAsyncTaskExecutor::Thread 53]  c.a.plugin.osgi.factory.OsgiPlugin Unable to start the plugin container for plugin 'com.atlassian.plugins.atlassian-remote-event-common-plugin'
```
I cannot find anything relevant that might help.
Bitbucket version 5.15.0
APM version: 1.2.0

I have also disabled the prometheus Exporter for this last test to ensure it is not interfering.

--- a/apm-agent-core/src/main/java/co/elastic/apm/agent/bci/OsgiBootDelegationEnabler.java
+++ b/apm-agent-core/src/main/java/co/elastic/apm/agent/bci/OsgiBootDelegationEnabler.java
@@ -22,9 +22,6 @@ package co.elastic.apm.agent.bci;
 import co.elastic.apm.agent.context.LifecycleListener;
 import co.elastic.apm.agent.impl.ElasticApmTracer;
 
-import java.util.Arrays;
-import java.util.List;
-
 /**
  * Required in OSGi environments like Equinox, which is used in WebSphere.
  * By adding the base package of the APM agent,
@@ -37,18 +34,25 @@ import java.util.List;
  * </p>
  */
 public class OsgiBootDelegationEnabler implements LifecycleListener {
-    private static final List<String> bootdelegationNames = Arrays.asList("org.osgi.framework.bootdelegation", "atlassian.org.osgi.framework.bootdelegation");
     private static final String APM_BASE_PACKAGE = "co.elastic.apm.agent.*";
+    // see https://confluence.atlassian.com/jirakb/using-javaagent-with-jira-790793295.html#UsingjavaagentwithJIRA-Resolution
+    private static final String ATLASSIAN_BOOTDELEGATION = "META-INF.services,com.yourkit,com.singularity.*,com.jprofiler," +
+        "com.jprofiler.*,org.apache.xerces,org.apache.xerces.*,org.apache.xalan,org.apache.xalan.*,sun.*,com.sun.jndi.*,com.icl.saxon," +
+        "com.icl.saxon.*,javax.servlet,javax.servlet.*,com.sun.xml.bind.*,co.elastic.apm.agent.*";
 
     @Override
     public void start(ElasticApmTracer tracer) {
-        for (String bootdelegationName : bootdelegationNames) {
-            final String systemPackages = System.getProperty(bootdelegationName);
-            if (systemPackages != null) {
-                System.setProperty(bootdelegationName, systemPackages + "," + APM_BASE_PACKAGE);
-            } else {
-                System.setProperty(bootdelegationName, APM_BASE_PACKAGE);
-            }
+        // may be problematic as it could override the defaults in a properties file
+        appendToSystemProperty("org.osgi.framework.bootdelegation", APM_BASE_PACKAGE);
+        appendToSystemProperty("atlassian.org.osgi.framework.bootdelegation", ATLASSIAN_BOOTDELEGATION);
+    }
+
+    private void appendToSystemProperty(String propertyName, String append) {
+        final String systemPackages = System.getProperty(propertyName);
+        if (systemPackages != null) {
+            System.setProperty(propertyName, systemPackages + "," + append);
+        } else {
+            System.setProperty(propertyName, append);
         }
     }
 

--- a/apm-agent-core/src/main/java/co/elastic/apm/agent/bci/OsgiBootDelegationEnabler.java
+++ b/apm-agent-core/src/main/java/co/elastic/apm/agent/bci/OsgiBootDelegationEnabler.java
@@ -22,6 +22,8 @@ package co.elastic.apm.agent.bci;
 import co.elastic.apm.agent.context.LifecycleListener;
 import co.elastic.apm.agent.impl.ElasticApmTracer;
 
+import javax.annotation.Nullable;
+
 /**
  * Required in OSGi environments like Equinox, which is used in WebSphere.
  * By adding the base package of the APM agent,
@@ -36,19 +38,23 @@ import co.elastic.apm.agent.impl.ElasticApmTracer;
 public class OsgiBootDelegationEnabler implements LifecycleListener {
     private static final String APM_BASE_PACKAGE = "co.elastic.apm.agent.*";
     // see https://confluence.atlassian.com/jirakb/using-javaagent-with-jira-790793295.html#UsingjavaagentwithJIRA-Resolution
-    private static final String ATLASSIAN_BOOTDELEGATION = "META-INF.services,com.yourkit,com.singularity.*,com.jprofiler," +
+    private static final String ATLASSIAN_BOOTDELEGATION_DEFAULTS = "META-INF.services,com.yourkit,com.singularity.*,com.jprofiler," +
         "com.jprofiler.*,org.apache.xerces,org.apache.xerces.*,org.apache.xalan,org.apache.xalan.*,sun.*,com.sun.jndi.*,com.icl.saxon," +
-        "com.icl.saxon.*,javax.servlet,javax.servlet.*,com.sun.xml.bind.*,co.elastic.apm.agent.*";
+        "com.icl.saxon.*,javax.servlet,javax.servlet.*,com.sun.xml.bind.*";
 
     @Override
     public void start(ElasticApmTracer tracer) {
         // may be problematic as it could override the defaults in a properties file
         appendToSystemProperty("org.osgi.framework.bootdelegation", APM_BASE_PACKAGE);
-        appendToSystemProperty("atlassian.org.osgi.framework.bootdelegation", ATLASSIAN_BOOTDELEGATION);
+        appendToSystemProperty("atlassian.org.osgi.framework.bootdelegation", ATLASSIAN_BOOTDELEGATION_DEFAULTS, APM_BASE_PACKAGE);
     }
 
-    private void appendToSystemProperty(String propertyName, String append) {
-        final String systemPackages = System.getProperty(propertyName);
+    private static void appendToSystemProperty(String propertyName, String append) {
+        appendToSystemProperty(propertyName, null, append);
+    }
+
+    private static void appendToSystemProperty(String propertyName, @Nullable String propertyValueDefault, String append) {
+        final String systemPackages = System.getProperty(propertyName, propertyValueDefault);
         if (systemPackages != null) {
             System.setProperty(propertyName, systemPackages + "," + append);
         } else {

--- a/apm-agent-core/src/test/java/co/elastic/apm/agent/bci/OsgiBootDelegationEnablerTest.java
+++ b/apm-agent-core/src/test/java/co/elastic/apm/agent/bci/OsgiBootDelegationEnablerTest.java
@@ -44,6 +44,7 @@ class OsgiBootDelegationEnablerTest {
         assertThat(System.getProperties())
             .containsEntry("org.osgi.framework.bootdelegation", "co.elastic.apm.agent.*")
             .containsKey("atlassian.org.osgi.framework.bootdelegation");
+        assertThat(System.getProperty("atlassian.org.osgi.framework.bootdelegation")).matches(".+,co.elastic.apm.agent.*");
     }
 
     @Test
@@ -53,5 +54,13 @@ class OsgiBootDelegationEnablerTest {
         assertThat(System.getProperties())
             .containsEntry("org.osgi.framework.bootdelegation", "foo.bar,co.elastic.apm.agent.*")
             .containsKey("atlassian.org.osgi.framework.bootdelegation");
+    }
+
+    @Test
+    void testAtlassianBootdelegationWithExistingProperty() {
+        System.setProperty("atlassian.org.osgi.framework.bootdelegation", "foo.bar");
+        osgiBootDelegationEnabler.start(mock(ElasticApmTracer.class));
+        assertThat(System.getProperties())
+            .containsEntry("atlassian.org.osgi.framework.bootdelegation", "foo.bar,co.elastic.apm.agent.*");
     }
 }

--- a/apm-agent-core/src/test/java/co/elastic/apm/agent/bci/OsgiBootDelegationEnablerTest.java
+++ b/apm-agent-core/src/test/java/co/elastic/apm/agent/bci/OsgiBootDelegationEnablerTest.java
@@ -43,7 +43,7 @@ class OsgiBootDelegationEnablerTest {
         osgiBootDelegationEnabler.start(mock(ElasticApmTracer.class));
         assertThat(System.getProperties())
             .containsEntry("org.osgi.framework.bootdelegation", "co.elastic.apm.agent.*")
-            .containsEntry("atlassian.org.osgi.framework.bootdelegation", "co.elastic.apm.agent.*");
+            .containsKey("atlassian.org.osgi.framework.bootdelegation");
     }
 
     @Test
@@ -52,6 +52,6 @@ class OsgiBootDelegationEnablerTest {
         osgiBootDelegationEnabler.start(mock(ElasticApmTracer.class));
         assertThat(System.getProperties())
             .containsEntry("org.osgi.framework.bootdelegation", "foo.bar,co.elastic.apm.agent.*")
-            .containsEntry("atlassian.org.osgi.framework.bootdelegation", "co.elastic.apm.agent.*");
+            .containsKey("atlassian.org.osgi.framework.bootdelegation");
     }
 }

